test/crowdsale/FinalizableCrowdsale.test.js

```diff
@@ -26,11 +26,10 @@ contract('FinalizableCrowdsale', function ([_, owner, wallet, thirdparty]) {
     this.closingTime = this.openingTime + duration.weeks(1);
     this.afterClosingTime = this.closingTime + duration.seconds(1);
 
-    this.token = await MintableToken.new();
+    this.token = await StandardToken.new([]);
```

Comment:
What is the `[]` argument?

Reply:
Probably the empty minters list from before I changed that token to a `StandardToken`, I find it worrying that truffle didn't complain about that.